python/src/aiconfig/eval/promptfoo/README.md

```diff
@@ -0,0 +1,23 @@
+# Promptfoo integration
+
+Use case: I'm a SWE who wants to run my AIConfig against a set of test cases specified in a config file. Each test case has the input and a success condition of my choosing.
+
+## Philosophy / design
+
+Prompfoo has a pretty nice interface (both input and outputs) for addressing the use case. Tests are specified in a yaml file and the test suite can be run with a simple command. The same config file makes it easy to connect your test suite to an AI config with a small amount of code.
+
+## How-to guide
+
+1. Write your test cases in a Promptfoo config file. See examples/travel/travel_promtfooconfig.yaml as an example.
+2. Define an AIConfig test suite settings file. It should have the prompt name and path to your aiconfig. See examples/travel/travel_aiconfig_test_suite_settings.json for example.
+3. Set your provider to point to run_aiconfig.py with your settings file as the argument. For e.g. see examples/travel/travel_promtfooconfig.yaml. Like this:
+
+```
+providers:
+  - exec:python ../../run_aiconfig.py ./travel_aiconfig_test_suite_settings.json
+```
+
+4. export your provider API key if needed so it's available to subprocess environments:
+   `export OPENAI_API_KEY=...`
+
+5. Run `npx promptfoo@latest eval -c path/to/promptfooconfig.yaml`
```

Comment:
But when you install aiconfig as a library would the path would be in `site-packages` or are you expecting the user to just copy/paste the run_aiconfig.py into their codebase? 

Even when developing aiconfig in dev, the paths can get annoying to deal with:
```
(aiconfig) suyogsonwalkar@Suyogs-MacBook-Pro-2 aiconfig % python -m site
sys.path = [
    '/Users/suyogsonwalkar/Projects/aiconfig',
    '/opt/homebrew/Caskroom/miniconda/base/envs/aiconfig/lib/python312.zip',
    '/opt/homebrew/Caskroom/miniconda/base/envs/aiconfig/lib/python3.12',
    '/opt/homebrew/Caskroom/miniconda/base/envs/aiconfig/lib/python3.12/lib-dynload',
    '/opt/homebrew/Caskroom/miniconda/base/envs/aiconfig/lib/python3.12/site-packages',
    '/Users/suyogsonwalkar/Projects/aiconfig/python/src',
]
USER_BASE: '/Users/suyogsonwalkar/.local' (exists)
USER_SITE: '/Users/suyogsonwalkar/.local/lib/python3.12/site-packages' (doesn't exist)
ENABLE_USER_SITE: True
```

Reply:
Not sure if that's the best solution, but it is one option. How else do you see someone who's using the library via `pip install python-aiconfig` using eval if they have a aiconfig.json and create a settings.json?

For the module version, I guess `python -m python-aiconfig.run_aiconfig_for_eval` or something to be specific right? Unless it can also be reused to just run with that settings file.